app/src/ui-blokada/res/values/strings_notification.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="notification_on_text">Notifications</string>
-    <string name="notification_on_description">When on, Blokada will display a notification every time something is blocked. Please note this is not the same as the VPN notification some devices show, and which cannot be turned off.</string>
+    <string name="notification_on_description">When on, Blokada will display a notification every time something is blocked. Please note this is not the same as the VPN notification some devices show, and which cannot be turned off.&lt;br&gt;You can configure which notifications should be displayed with the Settings below.</string>
```

Comment:
this: 
&lt;br&gt;
should be
&lt;br/&gt;

Reply:
Oh it encoded the characters in my comment. just add a slash before the second ampersand

Reply:
Fixed with 41a53ae20add8b5b199bdfd52b9a8b85edf2e5b5.